Tasks/AppCenterDistributeV3/Tests/L0EmptyBuildVersionDoesntAppearInBody.ts

```diff
@@ -74,37 +37,39 @@ nock('https://example.test')
         expiration_date: 1234567
     });
 
-//finishing symbol upload, commit the symbol 
-nock('https://example.test')
-    .patch("/v0.1/apps/testuser/testapp/symbol_uploads/100", {
-        status: 'committed'
-    })
-    .reply(200);
-
 // provide answers for task mock
 let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
     "checkPath" : {
-        "/test/path/to/my.apk": true,
+        "./test.apk": true,
     },
     "findMatch" : {
-        "/test/path/to/my.apk": [
-            "/test/path/to/my.apk"
+        "./test.apk": [
+            "./test.apk"
         ]
     }
 };
 tmr.setAnswers(a);
 
-fs.createReadStream = (s: string) => {
-    let stream = new Readable;
-    stream.push(s);
-    stream.push(null);
+mockAzure();
+
+mockFs();
```

Comment:
Could you add some reset for mocks? You can use library like simon for it (mock.reset()), or do it manually